packages/ui-extensions/src/surfaces/point-of-sale/components/CameraScanner/CameraScanner.ts

```diff
@@ -9,21 +9,43 @@ import {BannerVariant} from '../Banner/Banner';
 export interface CameraScannerBannerProps {
   /**
    * The title of the banner.
+   * @deprecated CameraScannerBannerProps has been deprecated. Use the CameraScannerNextProps instead.
```

Comment:
We don't need to mark all of these fields deprecated, I think just the one deprecation message on line 44 is good enough 😄